.github/workflows/bootstrap.yaml

```diff
@@ -0,0 +1,58 @@
+name: bootstrap
+
+on:
+  push:
+    branches:
+      - '*'
```

Comment:
Do we want to run this for _any_ branch (so also WIP ones), or just `master`?

Reply:
Using `*` means that at least for our own PRs we'll run the bootstrap tests before merge.